client/api.go

```diff
@@ -0,0 +1,35 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	v2netmap "github.com/nspcc-dev/neofs-api-go/v2/netmap"
+	rpcapi "github.com/nspcc-dev/neofs-api-go/v2/rpc"
+	"github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
+)
+
+// interface of NeoFS API server. Exists for test purposes only.
+type neoFSAPIServer interface {
+	netMapSnapshot(context.Context, v2netmap.SnapshotRequest) (*v2netmap.SnapshotResponse, error)
+}
+
+// wrapper over real client connection which communicates over NeoFS API protocol.
+// Provides neoFSAPIServer for Client instances used in real applications.
+type coreServer client.Client
```

Comment:
are there any preferences compared to `struct { client.Client }`?

Reply:
That's how we can underline that without tests we could live with `client.Client` only. Apart from that, no.

Reply:
ok, just dont like statements with too many parentheses that could be simplified: `rpcapi.NetMapSnapshot((*client.Client)(x), &req, client.WithContext(ctx))`